ucm2/USB-Audio/Realtek/ALC4080-HiFi.conf

```diff
@@ -88,6 +88,19 @@ If.asus-rog-usb {
 	}
 }
 
+If.wrx80-sage {
+	Condition {
+		Type RegexMatch
+		String "${CardComponents}"
+		Regex "USB(0b05:1984)"
+	}
+	True.Define {
+		Mic2Mixer "Analog In"
```

Comment:
Mic2Name is not defined, so it may be difficult for users to identify the physical jack. See `If.asus-rog-usb` block.

Reply:
Oh, yep I missed that. Thanks!